src/main/java/org/spongepowered/api/util/Axis.java

```diff
@@ -37,7 +37,7 @@
 
     private final Vector3d direction;
 
-    private Axis(final Vector3d vector3d) {
+    Axis(final Vector3d vector3d) {
```

Comment:
Keep these sort of OCD fixes out of PR's, make sure they're noted in the OCD #221 issue.